packages/gestalt/src/Popover.js

```diff
@@ -21,9 +21,9 @@ type Props = {|
    */
   color?: Color,
   /**
-   *
+   * Callback for key stroke events allowing keyboard navigation in Popover's childrens.
```

Comment:
nit: children is already plural, no need for the "s" suffix

Reply:
DONE